core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -1,30 +1,42 @@
 package chandu0101.scalajs.react.components
 
-import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 
+import japgolly.scalajs.react.Callback
```

Comment:
I'll do just a quick pass over the code. This is an improvement over what was there already, so LGTM before we even start :)